app/models/claim.rb

```diff
@@ -143,10 +143,6 @@ def payroll_gender_verified?
     verified_fields.include?("payroll_gender")
   end
 
-  def full_name
-    [first_name, middle_name, surname].compact.join(" ")
-  end
```

Comment:
Were you not leaving this in so @robbiepaul can use it?

Reply:
He said he didn't mind, but thought it was better not on Claim.

Reply:
Okay he does want it! :D I'll loose that commit.